contrib/mesos/pkg/scheduler/podtask/procurement.go

```diff
@@ -50,77 +62,212 @@ type AllOrNothingProcurement []Procurement
 
 // Procure runs each Procurement in the receiver list. The first Procurement func that
 // fails triggers T.Reset() and the error is returned, otherwise returns nil.
-func (a AllOrNothingProcurement) Procure(t *T, offer *mesos.Offer) error {
+func (a AllOrNothingProcurement) Procure(t *T, offer *mesos.Offer, n *api.Node, spec *Spec) error {
```

Comment:
OK, please capture this in the comments for the docs for the `Procurement.Procure` func.